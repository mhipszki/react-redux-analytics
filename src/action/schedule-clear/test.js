jest.useFakeTimers();

import scheduleClear from './';

test('generates action to schedule clearing user notification', () => {
  const action = scheduleClear();
  const dispatch = jest.fn();
  action(dispatch);
  jest.runAllTimers();
  expect(dispatch.mock.calls[0][0]).toMatchSnapshot();
});
