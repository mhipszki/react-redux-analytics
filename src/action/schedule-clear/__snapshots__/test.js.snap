// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generates action to schedule clearing user notification 1`] = `
Object {
  "type": "CLEAR_SENDING_INFO",
}
`;
